pom.xml

```diff
@@ -47,6 +47,17 @@
                 <enabled>true</enabled>
             </snapshots>
         </repository>
+        <repository>
```

Comment:
http://master-jenkins.skymind.io:8080/job/deeplearning4j/job/deeplearning4j/job/PR-4662/51/console says

```
...
[ERROR] Failed to execute goal on project deeplearning4j-modelexport-solr: Could not resolve dependencies
for project org.deeplearning4j:deeplearning4j-modelexport-solr:jar:1.0.0-SNAPSHOT: The following artifacts
could not be resolved: org.restlet.jee:org.restlet:jar:2.3.0, org.restlet.jee:org.restlet.ext.servlet:jar:2.3.0: Could
not find artifact org.restlet.jee:org.restlet:jar:2.3.0 in local-public (http://<redacted>/repository/local-public/) -> [Help 1]
...
```

which I think perhaps might be using .m2 cached results for previous build (without this pom.xml addition here) and given time when it retries (with this pom.xml addition here) it will perhaps (hopefully!) find the http://maven.restlet.com/org/restlet/jee/org.restlet/2.3.0/ stuff. Let's see.